### ISSUE TITLE
Support to handle pasted file when uploading file

### ISSUE THREAD

**Issue by EasonSoong** [payloadcms/payload#1423]
## Current Behavior

To paste file in the upload area didn't work.

![image](https://user-images.githubusercontent.com/57319837/202126904-22400c2b-ce8f-47ee-b7aa-92f8b06ee0c4.png)

## Expected Behavior

Support to handle pasted file in the upload area.

## Detailed Description

payload@1.1.26

**Comment by JarrodMFlesch** [comment]
Circling back here. We are revising the UI for this feature internally, once that is done we can implement the changes and merge into master. Thanks for your patience 👍
